R/MeasureSurvRCLL.R

```diff
@@ -77,15 +77,34 @@ MeasureSurvRCLL = R6::R6Class("MeasureSurvRCLL",
       # Bypass distr6 construction if underlying distr represented by array
       if (inherits(prediction$data$distr, "array")) {
         surv = prediction$data$distr
+        if (length(dim(surv)) == 3) {
+          # survival 3d array, extract median
+          surv = .ext_surv_mat(arr = surv, which.curve = 0.5)
+        }
         times = as.numeric(colnames(surv))
-        pdf = distr6:::cdfpdf(1 - surv)
```

Comment:
@bblodfon  This function automatically detects if surv is an array or matrix

Reply:
Detects yes, but does it convert to 2d matrix that we need for `C_Vec_WeightedDiscretePdf`? I think not, right? => https://github.com/xoopR/distr6/blob/main/R/transformers.R#L30